src/components/utils/ErrorHandler.js

```diff
@@ -0,0 +1,235 @@
+                    subheader={
+                        <Typography color="error">
+                            {getMessage("unexpectedError")}
+                        </Typography>
+                    }
+                />
+                <Divider oritentation="horizontal" />
+                <CardContent className={classes.cardContent}>
+                    <Grid container spacing={2} className={classes.container}>
+                        <ClientInfo baseId={baseId} />
+                    </Grid>
+                </CardContent>
+                <Divider oritentation="horizontal" />
+                <CardActions>
+                    <ContactSupport baseId={baseId} />
+                </CardActions>
+            </Card>
+        );
+    }
+    const errorResponse = errorObject.response;
+    if (errorResponse?.status === 401) {
+        return (
+            <Card>
+                <CardHeader
+                    title={
+                        <>
+                            <Typography component="span" variant="h5">
+                                <ErrorIcon color="primary" />
+                                {getMessage("signInReqd")}
+                            </Typography>
+                        </>
+                    }
+                />
+                <Divider orientation="horizontal" />
+                <CardContent className={classes.cardContent}>
+                    <Typography variant="h6" className={classes.signIn}>
+                        {getMessage("youMust")}
+                        <Link
+                            id={build(errBaseId, ids.SIGN_IN_LINK)}
+                            color="primary"
+                            onClick={() => {
+                                router.push(
+                                    `/${NavigationConstants.LOGIN}${router.asPath}`
+                                );
+                            }}
+                        >
+                            {getMessage("signIn")}
+                        </Link>
+                        {getMessage("viewThisPage")}
+                    </Typography>
+                    <Typography variant="subtitle2" className={classes.signIn}>
+                        {getMessage("needAccount")}
+                        <Link
+                            id={build(errBaseId, ids.REGISTER_LINK)}
+                            color="primary"
+                            onClick={() => {
+                                window.open(constants.USER_PORTAL);
+                            }}
+                        >
+                            {getMessage("register")}
+                        </Link>
+                    </Typography>
+                </CardContent>
+                <Divider orientation="horizontal" />
+                <CardActions>
+                    <ContactSupport baseId={baseId} />
+                </CardActions>
+            </Card>
+        );
+    } else {
+        return (
+            <Card>
```

Comment:
This `Card` is missing an `id`:
```suggestion
            <Card id={errBaseId}>
```

It seems the layout of every `Card` in these `if` checks are the same, and the only real difference are the `title`, `subheader`, and `CardContent` children. Personally I would try to refactor so there is only 1 `Card` return at the end of this component and set the `title`, `subheader`, and `CardContent` children using the if checks:
```jsx
        return (
            <Card id={errBaseId}>
                <CardHeader
                    title={headerTitle}
                    subheader={subHeader}
                />
                <Divider orientation="horizontal" />
                <CardContent className={classes.cardContent}>
                    {contents}
                </CardContent>
                <Divider orientation="horizontal" />
                <CardActions>
                    <ContactSupport baseId={errBaseId} />
                </CardActions>
            </Card>
        );
```

Just a suggestion, and not critical for merging this PR.